src/scenes/level.js

```diff
@@ -250,6 +257,47 @@ export class LevelScene extends Phaser.Scene {
                     console.log("Error: item does not have attack type");
                     break;
             }
+        } else if (this.levelType == CONSTANTS.LEVEL_TYPE.RESOURCE) {
+            let inventory = this.scene.get(CONSTANTS.SCENES.DASHBOARD).inventory;
+            let toolEquipped = false;
+            let i = -1;
+
+            switch (this.resourceType) {
+                case CONSTANTS.RESOURCES.WOOD:
+                    if (curWeapon.item == "Axe") {
+                        toolEquipped = true;
+                    } else {
+                        i = inventory.getKeywordInInventory("Axe");
+                    }
+                    break;
+                case CONSTANTS.RESOURCES.ORE:
+                    if (curWeapon.item == "Pickaxe") {
+                        toolEquipped = true;
+                    } else {
+                        i = inventory.getKeywordInInventory("Pickaxe");
+                    }
+                    break;
+                default:
+                    console.log("Error: this resource type does not exist.");
+                    break;
+            }
+
+            if (toolEquipped) {
+                imageName = curWeapon.sprite.texture.key + "-model";
+            } else {
+                let itemObj = getItemClass(
+                    characterData.getInventory()[i].item,
```

Comment:
Do you think we should add a function for this?

Reply:
This exact stuff should only be used for creating the animation so I'd say leave it